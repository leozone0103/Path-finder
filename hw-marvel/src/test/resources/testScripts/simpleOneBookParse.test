# Test the parser on a graph that contains two nodes
# with one edge each

LoadGraph heroes simpleData.csv
ListChildren heroes Daniel-hi-123
ListChildren heroes ZiangItsMe234