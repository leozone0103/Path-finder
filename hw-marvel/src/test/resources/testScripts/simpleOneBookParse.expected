# Test the parser on a graph that contains two nodes
# with one edge each

loaded graph heroes
the children of Daniel-hi-123 in heroes are: ZiangItsMe234(book1)
the children of ZiangItsMe234 in heroes are: Daniel-hi-123(book1)